blog/tutorial/01-index.md

```diff
@@ -33,7 +33,7 @@ Get started! The first step is to [install the `starport`](https://github.com/te
 After `starport` is installed, use it to create the initial app structure inside a directory named `blog`:
 
 ```
-starport app github.com/example/blog
+starport scaffold chain github.com/example/blog
 ```
 
 One of the main features of Starport is code generation. The command above has generated a directory structure with a working blockchain application. Starport can also add data types to your app with `starport type` command. To see it in action, follow the poll application tutorial. In this guide, however, you create those files manually to understand how it all works under the hood.
```

Comment:
```suggestion
One of the main features of Starport is code generation. The preceding command has generated a directory structure with a working blockchain application. You can also add data types to your app with the `starport type` command. To see code generation in action, follow the poll application tutorial. In this guide, however, we'll create those files manually to learn how Starport code generation works.
```

Reply:
thanks for these changes @Pantani  I know you didn't originally author this content, but we always strive to elevate our content. 
Here are some editorial suggestions: 
- instead of "above" let's use preceding, per style standards https://developers.google.com/style/word-list#letter-a 
- instead of having Starport be the actor, let's have the user "you" add the data types
- instead of "it" let's identify the subject "code generation" (unclear references are all over our docs) 
- I know it's tempting to use slang, but "under the hood" is colloquial, so can we just say "to learn..." ?